civismlext/preprocessing.py

```diff
@@ -123,10 +123,12 @@ class DataFrameETL(BaseEstimator, TransformerMixin):
         Can be a float or np.nan.
     dataframe_output : bool (default: False)
         If True, ETL output is a pd.DataFrame instead of a np.Array.
-    drop_null_cols : bool (default: False)
-        If True, columns which are all nulls will issue a warning during
-        `fit` and be dropped during `transform`. If False, there will not
-        be a check for null columns (but `fit` performance will be better).
+    drop_null_cols : {None, False, 'raise', 'warn'} (default: False)
```

Comment:
Are you happy with the name "drop_null_cols" now that it's not a boolean? I can see `False` and `"warn"` as options for a parameter named "drop_null_cols", but `"raise"` feels a bit stranger.

Reply:
Oops, missed this comment at first-- maybe "check_null_cols", or "handle_null_cols"? I think "check_null_cols" is clearest.